pkg/controllers/user/resourcequota/resource_quota_common.go

```diff
@@ -89,6 +145,13 @@ func getNamespaceResourceQuota(ns *corev1.Namespace) string {
 	return ns.Annotations[resourceQuotaAnnotation]
 }
 
+func getNamespaceContainerDefaultResourceLimit(ns *corev1.Namespace) string {
```

Comment:
can we use one function for `getNamespaceContainerDefaultResourceLimit`, `getProjectID` and `getNamespaceResourceQuota` with just passing the required annotation

Reply:
I'd leave it the way it is for now...want to try to avoid changing the existing code